.travis.yml

```diff
@@ -33,6 +33,7 @@ matrix:
   fast_finish: true
   allow_failures:
   - env: EMBER_TRY_SCENARIO=ember-release
+  - node_js: 'stable'
```

Comment:
This entry, and section, does not match https://github.com/ciena-frost/ember-frost-test/blob/master/.travis.yml, for example.  @juwara0 how/when are we handling the standardization of these configurations?  At the very least I'd rather not be introducing this change even if we're not going to remove the entire section for standardization (if that is the action we decided to take)